modules/custom/dumpsoftwareversions/main.nf

```diff
@@ -24,7 +24,7 @@ process CUSTOM_DUMPSOFTWAREVERSIONS {
     output:
     path "software_versions.yml"    , emit: yml
     path "software_versions_mqc.yml", emit: mqc_yml
-    path "versions.yml"             , emit: versions
+    path "versions.out.yml"         , emit: versions
```

Comment:
```suggestion
```

Reply:
Will need to change `meta.yml` too to remove this channel name.

Reply:
Done